### PR TITLE
Fix/get blobber

### PR DESCRIPTION
**Pull request by Kishan-Dhakan**
A brief description of the changes in this PR:
- Updates gosdk and refactors getblobbers

Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/zwalletcli/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- 0chain:
- Other: ...

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | fix/getblobbers  |
| zboxcli  | staging  |
| zwalletcli  | fix/get-blobber  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [cancelled] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | fix/getblobbers  |
| zboxcli  | staging  |
| zwalletcli  | fix/get-blobber  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

File: cmd/misc.go [hunk 1]
```diff
@@ -76,12 +78,38 @@ var getblobberscmd = &cobra.Command{
...
+			zcncore.GetBlobbers(statusBar, limit, offset, !active)
+			wg.Wait()
+
+			type nodes struct {
+				Nodes []*sdk.Blobber
+			}
+
+			var wrap nodes
+
+			err := json.Unmarshal([]byte(statusBar.errMsg), &wrap)
```
Comment (peterlimg) [inline cmd/misc.go:98]: Why the results are in the `statusBar.errMsg`, shouldn't it be returned by the `GetBlobbers()` directly.

**Review by peterlimg — CHANGES_REQUESTED** [review]

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | fix/getblobbers  |
| zboxcli  | staging  |
| zwalletcli  | fix/get-blobber  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Review by peterlimg — APPROVED** [review]

File: cmd/misc.go (hunk 1 above)
Comment (Kishan-Dhakan) [reply cmd/misc.go:98]: Well, yes, I will address this is in a separate. This refactoring was for pagination handling

**Pull request merged**